tools/rosbag/README.md

```diff
@@ -1,32 +0,0 @@
-# rosbag
```

Comment:
If this gets deleted, the only thing documenting this is the `-f` option, which only says `file with custom freq` at the moment.
As part of cleanup, we should probably rename this `--file-name` option to sth else

Reply:
We have the config file with info and at the time of recording if you do `rosbag record -h` .. you will get all options